pkg/resource/plugin/analyzer_config.go

```diff
@@ -0,0 +1,280 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+	"github.com/pulumi/pulumi/pkg/apitype"
+	"github.com/pulumi/pulumi/pkg/util/contract"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+// LoadPolicyPackConfigFromFile loads the JSON config from a file.
+func LoadPolicyPackConfigFromFile(file string) (map[string]AnalyzerPolicyConfig, error) {
```

Comment:
Somehow it feels odd that these are in the `plugin` package.  What was the reason for that?

Reply:
Yea, felt a bit odd to me as well. Moved all this out of `plugin` to it's own `analyzer` package.